turbo-cli/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "turbo-cli"
```

Comment:
I'm not sure what the right naming scheme here is, but this doesn't quite feel like a "cli".

Reply:
Can always rename it! The command is just `turbo`.